src/java.desktop/share/classes/javax/swing/plaf/metal/MetalLookAndFeel.java

```diff
@@ -145,9 +144,7 @@ static boolean isWindows() {
         if (!checkedWindows) {
             if (OSInfo.getOSType() == OSInfo.OSType.WINDOWS) {
                 isWindows = true;
-                @SuppressWarnings("removal")
-                String systemFonts = AccessController.doPrivileged(
-                    new GetPropertyAction("swing.useSystemFontSettings"));
```

Comment:
Missed the GetPropertyAction import for this file (can't annotate there).

Reply:
another one for a follow-up